src/ReverseProxy/Telemetry/ActivityPropagationHandler.cs

```diff
@@ -18,12 +19,16 @@ public sealed class ActivityPropagationHandler : DelegatingHandler
     {
         private const string RequestIdHeaderName = "Request-Id";
         private const string CorrelationContextHeaderName = "Correlation-Context";
+        private const string BaggageHeaderName = "Baggage";
```

Comment:
```suggestion
        private const string BaggageHeaderName = "baggage";
```

> In order to increase interoperability across multiple protocols and encourage successful integration, implementations SHOULD keep the header name lowercase.

we should use the lowercase header name as suggested by the spec